content/body/textbox.php

```diff
@@ -184,12 +184,12 @@
 
 
 
-<h2>Textbox With Character Counter</h2>
+<h2>Textbox with Character Counter</h2>
 
 <p>The character counter is visible at all times.  It is announced to screen reader users when:</p>
 
 <ol>
-  <li>They use the keyboard to access the textbox (e.g. using the TAB key).</li>
+  <li>They use the keyboard to access the textbox (e.g., using the TAB key).</li>
```

Comment:
I don't think we usually included commas after "e.g.". Do we need it here?
```suggestion
  <li>They use the keyboard to access the textbox (e.g. using the TAB key).</li>
```